contracts/strategies/strategy.sol.example

```diff
@@ -0,0 +1,125 @@
+/// @notice SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+/// @dev Import the appropriate base contract
+import "../strategy-joe-rush-farm-base.sol";
+
+/// @dev Contracts use NatSpec Format comments. NatSpec documentation is located here: https://docs.soliditylang.org/en/develop/natspec-format.html
+/// @notice The strategy contract for TraderJoe's Token1/Token2 Liquidity Pool with double rewards
+/// @dev If one of the tokens is a bridge extension token (ends with ".e"), the trailing E should be capitalized like so: StrategyJoeAvaxDaiE
+contract StrategyJoeToken1Token2 is StrategyJoeRushFarmBase {
+    /// @dev LP and Token addresses
+    uint256 public token1_token2_poolId = 10;
+    address public joe_token1_token2_lp = 0x1000000000;
+    
+    /// @dev Addresses for tokens not found in the base contracts
+    address public token2 = 0x2000000000;
+    address public reward1 = 0x300000000;
+
+    constructor(
```

Comment:
This is a function, so it needs a natspec header.  The header might look something like this:

`/// @notice Constructor
/// @param _governance The wallet which will be given ownership of this strategy
/// @param _strategist The wallet which will be given strategist role for this strategy
/// @param _controller The contract which will be set as the controller for this strategy
/// @param _timelock The contract acting as timelock for this strategy`

Reply:
That formatted poorly.  My descriptions above are generic placeholders since I am not deeply familiar with how these roles interact with the strategy.  Ideally the description should describe the interaction.